src/main/java/lotto/model/Lotto.java

```diff
@@ -0,0 +1,28 @@
+package lotto.model;
+
+import java.util.List;
+import lotto.util.validator.LottoNumbersValidator;
+
+public class Lotto {
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        LottoNumbersValidator.validateLottoNumbersSize(numbers);
+        LottoNumbersValidator.validateDuplicatedLottoNumbers(numbers);
+        this.numbers = numbers;
+    }
+
+    public boolean contains(int number) {
+        return numbers.contains(number);
+    }
+
+    public String showNumbers() {
+        return numbers.stream().sorted().toList().toString();
```

Comment:
컨벤션에서도 스트림 체인으로 이루어진 코드는 가독성을 위해 개행을 두는 것이 허용됩니다!
```suggestion
        return numbers.stream()
                .sorted()
                .toList()
                .toString();
```

Reply:
아하 감사합니다!😊